spot-client/src/common/css/button.scss

```diff
@@ -44,4 +44,37 @@
             background-color: lightcoral;
         }
     }
+
+    &.menu {
+        background-color: rgba(0, 0, 0, 0.5);
+        border: 1px solid white;
+        border-radius: 5px;
+        color: white;
+        font-size: 1em;
+        height: 48px;
+    }
+
+    &.volume {
+        background-color: rgba(0, 0, 0, 0.5);
+        height: 96px;
+        width: 96px;
```

Comment:
I've generally been using CSS variables for some sizes so that I can scale sizes across different device widths. Do you think it's too much? At some point I started feeling like it was unnecessary overhead.

Reply:
Right! Also modified the nav container a bit, because it didn't scale nicely with the new button. I hope it's good now